Makefile

```diff
@@ -85,13 +87,21 @@ go-version-check:
 		[ $(GO_VERSION_MAJ) -eq 1 -a $(GO_VERSION_MIN) -ge 12 ] || (echo "FATAL: Go version should be >= 1.12.x" ; exit 1 ; )
 
 .PHONY: lint
-lint:
+lint: deps
 	# explicitly enable GO111MODULE otherwise go mod will fail
 	GO111MODULE=on go mod tidy && GO111MODULE=on go mod vendor && GO111MODULE=on go mod verify
 	$(GO) vet ./...
 	test -z `$(GOFMT) -l $(SKUBA_SRCS)` || { $(GOFMT) -d $(SKUBA_SRCS) && false; }
 	$(TERRAFORM) fmt -check=true -write=false -diff=true ci/infra
-	find ci -type f -name "*.sh" | xargs bashate
+	$(BINPATH)/golangci-lint run cmd/... pkg/... internal/...
+	find ci -type f -name "*.sh" | xargs $(BINPATH)/shellcheck
```

Comment:
Wouldn't this make sense to be added on the `ci` `Makefile`, and maybe calling it from here?

Reply:
this is replacing current configuration, I prefer not to use this PR to do major changes